### DISCUSSION TITLE
Concerns about `get()`

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#90] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/41120635/160024669-cb5adacf-3884-4cda-b48c-a3e7ea625a9f.png)

Are we using it properly?

**Comment by jycouet** [comment, marked as answer]
I think so yes 🙃
I'm working with it in production in a "big" app, and have no issues so far.

**Reply by frederikhors** [reply]
I mean for use in apps with frequent patch on page change.

**Reply by jycouet** [reply]
You are worried because it's used in your `src/routes/continents/[code]/index.svelte` `load` function?
Really I use it a lot, and I never had issues. So I don't know what to say?!

**Reply by frederikhors** [reply]
Is there an alternative way?

**Reply by jycouet** [reply]
In the `context module` I don't think so.

Maybe you can think about `beforeNavigate` / `afterNavigate` and the the patch there? (with `$xxx.data`)
You can also patch it not in the load function, just in a `onMount` with `$xxx.data`

**Reply by frederikhors** [reply]
I saw you are using get also in the generated stores. I’m a bit worried about performances in a data heavy complex app. But I’ll try in one project an I’ll see. Keep working. Thanks a lot for your work!
